packages/blocks/src/surface-block/canvas-renderer/element-renderer/shape/utils.ts

```diff
@@ -50,7 +51,7 @@ export function drawGeneralShape(
 
   ctx.lineWidth = shapeModel.strokeWidth;
   ctx.strokeStyle = strokeColor;
-  ctx.fillStyle = fillColor;
+  ctx.fillStyle = filled ? fillColor : 'transparent';
```

Comment:
@L-Sun This has been missing before.

https://github.com/toeverything/blocksuite/blob/799772d22e6ed289e490893da3cf61d015f25bc2/packages/blocks/src/surface-block/canvas-renderer/element-renderer/shape/rect.ts#L77